backend/src/main/java/com/cs471/studentLoanSystem/common/login/LoginSystem.java

```diff
@@ -46,9 +46,9 @@
         }
         /* Found user but password doesn't match. Return relevant data but do not authenticate */
         if (!selectedUser.getPassword().equals(information.getPassword())) {
-            LoginResponse response = new LoginResponse();
-            response.setAuthenticated(false);
-            response.setRole(selectedUser.getRole());
+            LoginResponse response =
+                    new LoginResponse(true, selectedUser.getRole(), selectedUser.getId());
```

Comment:
I believe this is authenticating a user with incorrect password.

Reply:
Whoops, good catch. Fixed in latest commit